plugins/turing/go.mod

```diff
@@ -123,7 +127,10 @@ require (
 )
 
 replace (
+	github.com/caraml-dev/turing/engines/experiment => ../../../turing/engines/experiment
```

Comment:
I know this PR is WIP but this needs to be removed. :)

Reply:
Whoops thanks for highlighting that! :)